### PR TITLE
Add Llama2/Mistral/Mixtral test instructions for external users

### PR DESCRIPTION
**Pull request by RissyRan**
# Description

Add Llama2/Mistral/Mixtral test instructions for external users to address [issues/412](https://github.com/google/maxtext/issues/412):
* with a small change to suffix of tokenizer_path

### CODE REVIEW

**Comment by RissyRan** [comment]
Hi @A9isha, do you have the original source of Llama2 ckpt? is that from [here](https://llama.meta.com/llama-downloads/). Once you confirm, I could update the comments accordingly (if needed).

**Review by rwitten — APPROVED** [review]

**Comment by A9isha** [comment]
> Hi @A9isha, do you have the original source of Llama2 ckpt? is that from [here](https://llama.meta.com/llama-downloads/). Once you confirm, I could update the comments accordingly (if needed).

Yes correct, the user should get the original Llama2 ckpt from Meta and apply the conversion scripts.

File: end_to_end/test_llama2.sh
Comment (A9isha) [inline end_to_end/test_llama2.sh:1]: >#TODO(Training with Llama is not complete)
I think can now remove this

**Review by A9isha — APPROVED** [review]
Thanks Ran!

File: end_to_end/test_llama2.sh
Comment (RissyRan) [reply end_to_end/test_llama2.sh:1]: Removed :)

**Comment by RissyRan** [comment]
Hi team, is there anything blocking this merge? It's ready a couple days ago. I just rebase it again with the latest change. Thank you!

**Pull request merged**